src/test/java/net/sf/jabref/logic/importer/fetcher/MedlineFetcherTest.java

```diff
@@ -195,9 +173,15 @@ public void testEmptyEntryList() throws Exception {
         assertEquals(Collections.emptyList(), entryList);
     }
 
-    @Test(expected = FetcherException.class)
+    @Test
     public void testEmptyInput() throws Exception {
-        fetcher.performSearch("");
-        fail();
+        assertEquals(Collections.emptyList(),fetcher.performSearch(""));
+    }
+
+    private void cleanEntries(BibEntry entry){
+        entry.clearField(FieldName.ABSTRACT); //Remove abstract due to copyright
+        entry.clearField("copyright");
```

Comment:
Why are these `clearFields` are required (except abstract, which is of course ok)? The test should actually check that the fields are correctly deleted.